tuning/utils/merge_model_utils.py

```diff
@@ -102,3 +102,77 @@ def fetch_base_model_from_checkpoint(checkpoint_model: str) -> str:
             "Base model adapter config exists, but has no base_model_name_or_path!"
         )
     return adapter_dict["base_model_name_or_path"]
+
+def _copy_files_to_directory(src: str , dest: str, exclude_files:list[str] = []):
+    import shutil
+    src_files = os.listdir(src)
+    for file_name in src_files:
+        if file_name in exclude_files:
+            continue
+        full_file_name = os.path.join(src, file_name)
+        if os.path.isfile(full_file_name):
+            shutil.copy(full_file_name, dest)
+
+def post_process_vLLM_adapters_new_tokens(path_to_checkpoint: str, modified_checkpoint_path:str = None):
```

Comment:
I think its a good idea to document down 
1. why we need this function
2. what this function tries to do, and description of the rough logic.